build/checkstyle.xml

```diff
@@ -48,6 +48,9 @@
     <module name="EmptyStatement"/>
     <module name="SuperFinalize"/>
     <module name="EqualsHashCode"/>
+    <module name="EmptyCatchBlock">
+      <property name="exceptionVariableName" value="ignore|ignored"/>
```

Comment:
What's the benefit we get in forcing a naming of variable on empty try-catch? I think this is unnecessary strict.